azure-pipelines.yml

```
@@ -300,19 +300,22 @@ jobs:
       pwd
       ls -l
       if [[ "$OS_NAME" == "Linux" ]]; then
-        CC=$(Compiler)-$(CompilerVer)
+        SELECTED_CC=$(Compiler)-$(CompilerVer)
         if [[ "$(Compiler)" == "clang" ]]; then
-          CXX=clang++-$(CompilerVer)
-          PACKAGES="$CC $PACKAGES"
+          SELECTED_CXX=clang++-$(CompilerVer)
+          PACKAGES="$SELECTED_CC $PACKAGES"
         else
-          CXX=g++-$(CompilerVer)
-          PACKAGES="$CXX $PACKAGES"
+          SELECTED_CXX=g++-$(CompilerVer)
+          PACKAGES="$SELECTED_CXX $PACKAGES"
         fi
+      else
+        SELECTED_CC=$(Compiler)
+        SELECTED_CXX=$(Compiler)
```

@inikep I believe here we need `++` for `CXX`
may be just
```suggestion
        SELECTED_CC=gcc
        SELECTED_CXX=g++
```

---

I think `clang` and `clang++` is more precise.